worlds/messenger/Constants.py

```diff
@@ -0,0 +1,153 @@
+# items
+# listing individual groups first for easy lookup
+NOTES = [
+    "Key of Hope",
+    "Key of Chaos",
+    "Key of Courage",
+    "Key of Love",
+    "Key of Strength",
+    "Key of Symbiosis"
+]
+
+PROG_ITEMS = [
+    "Wingsuit",
+    "Rope Dart",
+    "Ninja Tabi",
+    "Power Thistle",
+    "Demon King Crown",
+    "Ruxxtin's Amulet",
+    "Fairy Bottle",
+    "Sun Crest",
+    "Moon Crest",
+    # "Astral Seed",
+    # "Astral Tea Leaves",
+]
+
+PHOBEKINS = [
+    "Necro",
+    "Pyro",
+    "Claustro",
+    "Acro"
+]
+
+USEFUL_ITEMS = [
+    "Windmill Shuriken"
+]
+
+# item_name_to_id needs to be deterministic and match upstream
+ALL_ITEMS = [
+    *NOTES,
+    "Windmill Shuriken",
```

Comment:
just a warning - since you enumerate ALL_ITEMS for the name_to_id, new items will have to be appended to the end, so you can not add extra notes or phobekins without breaking backwards compat. This is the exact mistake that DS3 made.

This should be solved by using checksum based data package, but we are not there yet. After the next release, we still need to update all clients to use the checksum instead of versions.